modules/monitoring/templates/govuk-uptime-collector.conf.erb

```diff
@@ -0,0 +1,6 @@
+description "govuk-uptime-collector"
+
+start on runlevel [2345]
+respawn
+
+exec /usr/lib/rbenv/versions/2.3.1/bin/ruby /usr/local/bin/govuk_uptime_collector <%= @environment %> publishing-api content-store
```

Comment:
Do other processes specify a ruby version like this? it feels a bit precarious for when we end usage of 2.3.1. 

I wonder if there's a way we can execute through rbenv to specify we just want a 2.x version of Ruby to run it?

Reply:
I couldn't figure out a way of pinning to any version of Ruby 2.x, but I was able to use rbenv to select 2.3 instead of 2.3.1 specifically.